### PR TITLE
build02: backup nixpkgs-update logs

### PR DESCRIPTION
**Pull request by zowoq**
Closes https://github.com/nix-community/infra/issues/541

@Mic92 I copied most of this from your dotfiles as I noticed that you used to have a borgbackup storagebox.

cc @nix-community/admin

### CODE REVIEW

File: build02/nixpkgs-update-backup.nix [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  systemd.services.borgbackup-job-nixpkgs-update.serviceConfig.ReadWritePaths = [
+    "/var/log/telegraf"
+  ];
+
+  services.borgbackup.jobs.nixpkgs-update = {
+    paths = [
+      "/var/log/nixpkgs-update"
+    ];
+    repo = "u348918@u348918.your-storagebox.de:/./nixpkgs-update";
+    encryption.mode = "none";
```
Comment (zowoq) [inline build02/nixpkgs-update-backup.nix:23]: I don't think we need to bother with encryption?

**Comment by Mic92** [comment]
Do you have full access to this storage box or do you need further credentials / setup from my side?

File: build02/nixpkgs-update-backup.nix (hunk 1 above)
Comment (Mic92) [reply build02/nixpkgs-update-backup.nix:23]: Probably yes since the logs are public.

**Comment by zowoq** [comment]
Shouldn't require anything else, I was able to set a password and copy the ssh key.

**Pull request merged**